wandb/sdk/lib/wburls.py

```diff
@@ -33,6 +33,7 @@ def _get_urls(self) -> Dict["URLS", str]:
             upgrade_local="https://wandb.me/local-upgrade",
             multiprocess="http://wandb.me/init-multiprocess",
             wandb_init="https://wandb.me/wandb-init",
+            deployments="https://wandb.me/deployments",
```

Comment:
Can we change the path to "https://wandb.me/wandb-server". I've created a wandb.me link for it already to direct users to the wandb server quickstart docs.

Reply:
I have changed the link